ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -242,7 +242,14 @@ module.exports = Marionette.LayoutView.extend({
         .done((data, textStatus, jqxhr) => {
             _this.model.set({
                 type: 'custom'
-            })
+            });
+            const preferences = _user.getQuerySettings();
```

Comment:
This fixes when creating a new search form, it resets the query preference type instead of being stuck on `new-form`

Reply:
👍 This isn't you but I wonder why we redefine user to _user.

Reply:
Pretty sure that was old code that just never got removed before being merged.